_posts/2015-07-17-cassandra-write-performance-on-gce-and-aws.md

```diff
@@ -5,4 +5,91 @@ layout: post
 comments: true
 ---
 
-... or how we learned to create cross-cloud repeatable benchmarks for everybody. 
+*... or how we created cross-cloud repeatable benchmarks for everybody.*
```

Comment:
tunable* sorry, missed that. Otherwise people will just say this is pts/perfkit etc